boule/_realizations.py

```diff
@@ -290,10 +289,10 @@
     reference=(
         "Semi-axis: Corlies, P., et al. (2017). Titan’s Topography and Shape at "
         "the End of the Cassini Mission. Geophysical Research Letters, 44(23), "
-        "11,754-11,761. https://doi.org/10.1002/2017GL075518; "
+        "11,754-11,761. https://doi.org/10.1002/2017GL075518\n"
```

Comment:
What if we make this line breaks more explicit by moving them to their own lines?
```suggestion
        "11,754-11,761. https://doi.org/10.1002/2017GL075518"
        "\n"
```